### PR TITLE
Make recurse logic public

### PR DESCRIPTION
**Pull request by phanimarupaka**
@mortent 

This PR is to make RecurseSubpackages logic public so that upstream kpt can use it.

### CODE REVIEW

File: cmd/config/util/util.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Copyright 2019 The Kubernetes Authors.
 // SPDX-License-Identifier: Apache-2.0
 //
-package commands
+package util
```
Comment (monopole) [inline cmd/config/util/util.go:4]: `util`, `misc` should be a last resort.

How about `shell` or `runner`?

The methods appear to be
```
ExecuteCmd
ExecuteCmdOnPkgs
CmdRunner
FixDocs
HandleError
```

Comment (phanimarupaka) [reply cmd/config/util/util.go]: @monopole Thanks for the comment. I renamed it to `runner` which is more appropriate.

File: kyaml/fn/runtime/starlark/example_test.go [hunk 2]
```diff
@@ -131,7 +131,7 @@ spec:
   value: "hello world"
 `)
 	if err != nil {
-		log.Fatal(err)
+		log.Println(err)
```
Comment (mortent) [inline kyaml/fn/runtime/starlark/example_test.go:134]: Why are we changing this?

Comment (phanimarupaka) [reply kyaml/fn/runtime/starlark/example_test.go:134]: Fatal does os.Exit which will not perform defer cleanup. Lint caught that.

**Pull request merged**